dashboard/test/helpers/delete_accounts_helper_test.rb

```diff
@@ -914,6 +914,55 @@ def run(*_args, &_block)
     assert_logged 'Cleared 2 CodeReviewComment'
   end
 
+  #
+  # Table: dashboard.code_review_requests
+  # Table: dashboard.code_review_notes
+  #
+  test "soft deletes comments for purged user" do
```

Comment:
Nit: maybe an update to this test name that notes there's some hard deleting going on this case, and aligns with the naming in the next test? Maybe "deletes code review comments on reviews owned by purged user"

Reply:
good call